### PR TITLE
[bitnami/mysql] add startupProb

### LINKED ISSUE #6444: [mysql] add startupProb
**Which chart**:
Mysql

**Is your feature request related to a problem? Please describe.**
Mysql can take too long to startup, depending on Hard drive, init script, and other cases, example:
https://github.com/bitnami/bitnami-docker-mysql/issues/121

**Describe the solution you'd like**
Having a startup probe would improve the experience without damaging the the other probes

https://kubernetes.io/docs/tasks/configure-pod-container/configure-liveness-readiness-startup-probes/#define-startup-probes

### PR DESCRIPTION
**Pull request by yelhouti**
Signed-off-by: Youssef El Houti <youssef.elhouti@gmail.com>

**Description of the change**

**Benefits**

**Possible drawbacks**

**Applicable issues**

  - fixes #6444

**Additional information**

**Checklist** 

- [X] Chart version bumped in `Chart.yaml` according to [semver](http://semver.org/).
- [X] Variables are documented in the README.md
- [X] Title of the PR starts with chart name (e.g. [bitnami/<name_of_the_chart>])

### CODE REVIEW

File: bitnami/mysql/templates/primary/statefulset.yaml [hunk 1]
```diff
@@ -207,6 +207,21 @@ spec:
           {{- else if .Values.primary.customReadinessProbe }}
           readinessProbe: {{- include "common.tplvalues.render" (dict "value" .Values.primary.customReadinessProbe "context" $) | nindent 12 }}
           {{- end }}
+          {{- if .Values.primary.startupProbe.enabled }}
```
Comment (Mauraza) [inline bitnami/mysql/templates/primary/statefulset.yaml:210]: With the `customStartupProbe` is not working, could you fix it?

File: bitnami/mysql/templates/secondary/statefulset.yaml [hunk 2]
```diff
@@ -193,6 +193,21 @@ spec:
           {{- else if .Values.secondary.customReadinessProbe }}
           readinessProbe: {{- include "common.tplvalues.render" (dict "value" .Values.secondary.customReadinessProbe "context" $) | nindent 12 }}
           {{- end }}
+          {{- if .Values.secondary.startupProbe.enabled }}
```
Comment (Mauraza) [inline bitnami/mysql/templates/secondary/statefulset.yaml:196]: the same as the primary

**Review by Mauraza — CHANGES_REQUESTED** [review]
Hi @yelhouti,

Thanks for the PR!!, could you check the comments?

File: bitnami/mysql/templates/primary/statefulset.yaml (hunk 1 above)
Comment (yelhouti) [reply bitnami/mysql/templates/primary/statefulset.yaml:210]: @Mauraza I used exactly the same code as readiness... what do you mean by not working ?

Comment (Mauraza) [reply bitnami/mysql/templates/primary/statefulset.yaml:210]: I missed change startupProbe.enabled to false, it's working

File: bitnami/mysql/Chart.yaml [hunk 3]
```diff
@@ -25,4 +25,4 @@ name: mysql
 sources:
   - https://github.com/bitnami/bitnami-docker-mysql
   - https://mysql.com
-version: 8.5.10
+version: 8.5.11
```
[the remark this answers is not in the document]
Comment (Mauraza) [reply bitnami/mysql/Chart.yaml:28]: This is pending

**Review by Mauraza — CHANGES_REQUESTED** [review]

**Review by Mauraza — APPROVED** [review]
Hi @yelhouti,

Thanks again for the PR!!, LGTM!!

**Pull request merged**